### PR TITLE
Add profile page for internal users

### PR DESCRIPTION
**Pull request by dooven**
Changes
- Added profile page for internal users
- When users click on mentions in Discover & Timeline pages, the profile page is also gets shown
Peek 2020-09-12 17-37

image

Screenshot from 2020-09-12 17-39-21

Screenshot from 2020-09-12 17-39-54

### CODE REVIEW

**Comment by dooven** [comment]
Fix the follow/unfollow icon

**Review by prologic — APPROVED** [review]
LGTM! Very impressive! 🎉

File: lib/api.dart [hunk 1]
```diff
@@ -54,6 +56,20 @@ class Api {
     return user;
   }
 
+  Future<User> loginUsingCachedData() async {
+    var _user = await user;
+
+    final profileReponse =
```
Comment (aaadonai) [inline lib/api.dart:62]: missspelling of Response

**Review by aaadonai — APPROVED** [review]

File: lib/api.dart (hunk 1 above)
Comment (dooven) [reply lib/api.dart]: Thanks for spotting it :eye:

**Pull request merged**